diagrams/sequence-diagrams/changing-a-form.md

```diff
@@ -0,0 +1,3 @@
+# Changing a form
+
+TODO
```

Comment:
Could we leave this file out for the moments? It feels like the current process is already documented as part of the "Set other form details" workflow. I do think its needed but only when we come to do draft/live versioning.  This new draft/live process will be slightly different from the existing "Set other form details". 

I'll branch off this PR in the meantime and try document what draft/live versioning might look like

Reply:
I've added something that reflects current (temporary) behaviour. A future PR can show proposed behaviour.